lib/partition_setup.pm

```diff
@@ -122,17 +122,17 @@ sub addpart {
             wait_still_screen 1;
             send_key((is_storage_ng) ? 'alt-f' : 'alt-s');
             wait_screen_change { send_key 'home' };    # start from the top of the list
-            assert_screen 'partition-selected-ext2-type', timeout => 10;    # 1. fs in the list
+            assert_screen(is_sle('<15') ? 'partition-selected-btrfs-type' : 'partition-selected-ext2-type'), timeout => 10;
```

Comment:
I believe we should use is_storage_ng flag here, as we call addpart method on TW as well. Actually, it also depend on role of the partition: https://openqa.opensuse.org/tests/687903 so here XFS will be default. So to be honest I would rather try wait_still_screen as generic needle won't work either as we will create 2 or 3 with different filesystems.

Reply:
Oh thanks, is_storage_ng flag makes more sence to be used over here. IMHO needle option looks more feasible since it is not about what we are choosing, but how the list of FSes is ordered. Where storage-ng list starts with Ext2, and old stack with btrfs. As we are encountering send-key issues on daily basis it might be useful to check whether we are or we aren't on the top of the list.